latest/Dockerfile.rhel7

```diff
@@ -28,7 +28,8 @@ LABEL summary=$SUMMARY \
       io.openshift.tags="database,postgresql,postgresql96,rh-postgresql96" \
       name="rhscl/postgresql-96-rhel7" \
       com.redhat.component="rh-postgresql96-docker" \
-      version="1"
+      version="9.6" \
+      release="1"
```

Comment:
We are not setting release in newer images anymore as it is updated by the build system.

Reply:
Also using version="1" for all new images, since the version of postgresql is already incorporated into the image name

Reply:
Could you please specify which images are we discussing? Are they new release of old images? Could it be safe to change it in all postgresql version images or is this only related to this(postgresql 9.6 rhel) image?

Reply:
I meant new images introduced in RHSCL 3.0 (and later) - so for this repository that would be only postgresql 9.6.

Changing the version and release for older images would likely not introduce any major issues but it would be confusing to see the version drop to 1 so we decided to leave the older images as is.